app/controllers/concerns/headers.rb

```diff
@@ -0,0 +1,14 @@
+# Concern which has methods dealing with headers that might be interesting for
+# controllers deriving directly from ActionController::Base.
+module Headers
+  extend ActiveSupport::Concern
+
+  included do
+    before_action :default_headers
```

Comment:
maybe `after_action`? I know that it doesn't make any difference in the end but `after` seems to be more related to the response.